### DISCUSSION TITLE
How to make ONC File containing Proxy Settings

### DISCUSSION THREAD

**Discussion by HackrTP** [3kh0/ext-remover#1159] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE]
I am wondering how to edit the ONC files option for proxy to be Manual and add thee hostname, Port, username and password. If this can be done, it will become extremely easy to bypass most if not all web filters on school networks. If anyone has information please let me know as I have been trying to make one but keep getting a ONC File parse error and it doesn't update the settings for proxy.

**Comment by S-PScripts** [comment]
dupe of https://github.com/3kh0/ext-remover/discussions/959
